Observer/Backend/AddCatalogToCart.php

```diff
@@ -49,11 +73,7 @@ public function execute(Observer $observer)
                      __('added to queue')
                  );
              } catch (\Exception $e) {
-                 $this->logger->critical(
-                     $e->addSuccess(
-                     __('queue not added')
-                 )
-                 );
+                 $this->_messageManager->addError($e->getMessage());
```

Comment:
@NandhiniKarthi I see you're using the deprecated methods here (addErrorMessage). Please run your module with Magento coding standard tool and fix the issues along with replace all deprecated functions